shibata added k8s upgrade and Vantiq_resources_introduction (#32)
fujitake/vantiq-related: vantiq-platform-operations/readme.md

- [Procedure for generating a CSR for a server certificate](./docs/eng/prepare_csr4rsasslcert.md)
- [Vantiq Cloudwatch Logs](./docs/eng/vantiq-cloudwatch.md)  
- [Procedure for tearing down Vantiq Private Cloud](./docs/eng/vantiq-teardown.md)

## List of maintenance items for the Vantiq platform

|# | Items  | Description | Appropriate timing | Standard preparation time | Service outage on update | Operator|
|---|----|-------|------|----------------|----------------------|----------|

> |1  | Vantiq License Renewal  | The license file referenced by the Vantiq cluster. **If the license is not valid, the Vantiq service will stop.**| From after the PO is issued to before the license expires.   | 1 week         | Not required| Vantiq Support|

Reviewer: ```suggestion
|1  | Vantiq License Renewal  | The license file referenced by the Vantiq cluster. **If the license is not valid, the Vantiq service will stop.**| After the PO has been issued but before the license expires.   | 1 week         | Not required| Vantiq Support|
```

> |2  | SSL Certificate Renewal  | SSL certificate to be used by the Vantiq cluster.<br />**When it expires, it will not possible to log in to the Vantiq IDE or connect to the service via HTTPS REST.** | Before the certificate expires.  | 2 weeks        | Not required | Vantiq Support|

Reviewer: ```suggestion
|2  | SSL Certificate Renewal  | SSL certificate to be used by the Vantiq cluster.<br />**If it expires, one cannot log in to the Vantiq IDE or connect to the service via HTTPS REST.** | Before the certificate expires.  | 2 weeks        | Not required | Vantiq Support|
```

|3  | Vantiq Minor Upgrade   | Version upgrade of Vantiq with the addition of features. | About once every four months (three times a year).   | 1 week         | Required   | Vantiq Support|

> |4  | Vantiq Patch Upgrade | Version upgrade of the Vantiq to fix issues of the current version.  | At any time. At the time of release of fixing issues that interfere with operations. | 2 days | Not required  | Vantiq Support|

Reviewer: ```suggestion
|4  | Vantiq Patch Upgrade | Version upgrade of the Vantiq to fix issues of the current version.  | At any time the fix that impacts the normal operations is released. | 2 days | Not required  | Vantiq Support|
```

> |5  | Service Principal Account Update | (Only when configuring Internal Load Balancer in Azure)<br />Service Principal that has authority on AKS is used in order to configure Vantiq as Private.<br />**The service may stop after the expiration date.** | Before the Service Principal expires. | 1 week         | Required  | Vantiq Support|

Reviewer: ```suggestion
|5  | Service Principal Account Update | (Only when configuring Internal Load Balancer in Azure)<br />Service Principal used in AKS that has a permission to configure Load Balancer as Private.<br />**The service may stop after the expiration date.** | Before the Service Principal expires. | 1 week         | Required  | Vantiq Support|
```

Reviewer: レビューが終わったので修正してください。
完了したらお知らせください。

Author: commit をしました。
200 近い修正で大変お手数をおかけしてしまいました。
ありがとうございます。
